caiman/components_evaluation.py

```diff
@@ -414,20 +415,11 @@ def evaluate_components(Y, traces, A, C, b, f, final_frate, remove_baseline=True
 
 
 #%%
-# FIXME xrange is python2-specific
```

Comment:
This function wasn't used anywhere. Fixing it would've been easy for python3, but no point if nobody's using it.